rollup.config.mjs

```diff
@@ -44,6 +44,18 @@ const __filename = fileURLToPath(import.meta.url)
 const projectRootDir = dirname(__filename)
 
 const plugins = [
+  {
+    transform(src, id) {
+      if (id.includes('rollup-plugin-vue=styles')) {
+        const split = src.split('/* STYLES STRIP IMPORTS MARKER */')
```

Comment:
Comment about the location in the code (full path to the `styles.scss` file) and the reason why we're doing this would be helpful.